### PR TITLE
Glitch out of jail protection + advanced log

### PR DESCRIPTION
**Pull request by SimZor**
When a player glitches through walls out of prison they will get
teleported back, get a notice and they will be logged into RPT.

### CODE REVIEW

**Comment by yeiij** [comment]
I think is a good idea allow player escape in heli, role: friend rescue him

**Comment by SimZor** [comment]
Yeiij, this will allow them to escape the jail with a Helicopter. They will get teleported back to the jail if they're not in a Helicopter. :)

Don't know if that is what you meant.

**Comment by yeiij** [comment]
Oki, didnt saw "!" good job.

File: Altis_Life.Altis/core/civilian/fn_jailMe.sqf [hunk 1]
```diff
@@ -36,6 +36,24 @@ for "_i" from 0 to 1 step 0 do {
         hintSilent parseText format [(localize "STR_Jail_Time")+ "<br/> <t size='2'><t color='#FF0000'>%1</t></t><br/><br/>" +(localize "STR_Jail_Pay")+ " %3<br/>" +(localize "STR_Jail_Price")+ " $%2",_countDown,[life_bail_amount] call life_fnc_numberText,if (life_canpay_bail) then {"Yes"} else {"No"}];
     };
 
+    if (!(vehicle player isKindOf "air") && (player distance (getMarkerPos "jail_marker") > 50) && (FETCH_CONST(life_adminlevel) < 1)) then {
+
+        // -- Teleport back to jail and notify the player
+        player setPos (getMarkerPos "jail_marker");
+        hint "You glitched yourself out of jail, this has been logged. The Staff Team has been notified.";
```
Comment (Jawshy) [inline Altis_Life.Altis/core/civilian/fn_jailMe.sqf]: All [hints](https://community.bistudio.com/wiki/hint) should be [localised](https://community.bistudio.com/wiki/localize).

File: Altis_Life.Altis/core/civilian/fn_jailMe.sqf [hunk 2]
```diff
@@ -36,6 +36,24 @@ for "_i" from 0 to 1 step 0 do {
...
+
+        // -- Log to RPT
+        if (LIFE_SETTINGS(getNumber,"player_advancedLog") isEqualTo 1) then {
+            if (LIFE_SETTINGS(getNumber,"battlEye_friendlyLogging") isEqualTo 1) then {
+                money_log = format [localize "STR_DL_JL_glitchjail_BEF",player getVariable ["realname",name _unit]];
+            } else {
+                money_log = format [localize "STR_DL_JL_glitchjail",profileName,(getPlayerUID player),player getVariable ["realname",name _unit]];
```
Comment (Jawshy) [inline Altis_Life.Altis/core/civilian/fn_jailMe.sqf]: Why [`money_log`](https://github.com/AsYetUntitled/Framework/blob/7784f81aac515a5670cb7379cf367c117665e00c/Altis_Life.Altis/config/Config_Master.hpp#L15)? This has nothing to do with money. It should be [`advanced_log`](https://github.com/AsYetUntitled/Framework/blob/7784f81aac515a5670cb7379cf367c117665e00c/Altis_Life.Altis/config/Config_Master.hpp#L14).

Comment (Jawshy) [inline Altis_Life.Altis/core/civilian/fn_jailMe.sqf]: The comment for Config_Master.hpp → `player_advancedLog` should include "escaping jail."

**Comment by fresqo** [comment]
The helicopter breakout is soooo 2014

**Comment by SimZor** [comment]
As you mentioned it should be advanced_log but forgot to change it. It was my intention from the beginning. 

I'll also localise the hints

**Comment by SimZor** [comment]
Changed everything that people pointed out.

**Comment by BoGuu** [comment]
Or we just disable running and/or jumping?

**Comment by SimZor** [comment]
I mean sure, but it doesn't hurt to have this as extra protection?

**Comment by BoGuu** [comment]
And for the people that have a jail that doesn't use helicopters, or perhaps a bigger jail? Do you not think some options to reduce their movement capabilities would be better?

**Comment by SimZor** [comment]
Maybe, sure.

**Comment by SimZor** [comment]
Maybe make a option to toggle this off/on and also make a breakout script that can be toggled off / on.

I'll do that as soon as I get a chance to.

**Comment by Latouth** [comment]
Oh, people still use helis to breakout?
I made mine so you need a water vehicle that also can drive on land (Makes it super slow for them to breakout, more realistic) then they have to drive (or "float"?) over to the Water Prison, then ye... lol

**Comment by BoGuu** [comment]
Closing, it's been a while. If you want to redo it, as you said, follow what I said in my last comment, as to account for every situation.

**Pull request closed without merging**